Buildings/Resources/Scripts/OpenModelica/compareVars/Buildings.Experimental.DHC.Examples.Cooling.CoolingSystem.mos

```diff
@@ -0,0 +1,14 @@
+compareVars :=
+  {
+    "pla.port_a.p",
+    "pla.port_b.p",
+    "bld[1].p_rel",
```

Comment:
Please verify `nBui` is consistent after update in this file as well.

Reply:
CompareVars updated.